python/lsst/pipe/base/cmdLineTask.py

```diff
@@ -429,6 +429,17 @@ def __call__(self, args):
                 exitStatus=exitStatus,
             )
 
+    def _runDataRef(self, task, dataRef, kwargs):
```

Comment:
This method needs a docstring.
I suggest naming it `runTask`.
I'm not a fan of the leading underscore on the name, which tells the user that this very useful method should never be overridden, which I think is a bit much to ask.

Reply:
Accepted and changed throughout.  This makes the LegacyTaskRunner clearer, as it never called runDataRef().